README file for the examples directory + years corrections

MikhailRyazanov opened the pull request:
Addressing #321: a short README.txt file in the `examples` directory to clarify that the scripts should be run _there_.

And minor corrections to the v0.8.4 release information:
* Updating the copyright years.
* Correcting the release date (was 2020 instead of 2021) in the changelog.

MikhailRyazanov commented:
Where is the Travis CI check?!

DanHickstein commented:
This looks good to me!

But I'm not sure why Travis isn't working. Maybe we can go ahead and merge this one and open a new PR for testing/fixing the Travis build.

DanHickstein commented:
I checked on Travis-CI. We are getting an error saying: "Could not authorize build request for PyAbel/PyAbel", which I think basically means that our "subscription expired". My guess is that, when we moved from travis-ci.org to travis-ci.com, we somehow got switched from an open-source project to a trail commercial subscription and this trial ran out. I emailed them to see if they can fix it. Open source projects are supposed to be free-of-charge.

DanHickstein commented:
Okay, it looks like we weren't signed up for any of their plans. I signed us up for the "free" plan, which gives us 10,000 free minutes of build time. Since we only use a few minutes per month, this should last for many years. If we require more minutes, then we can just ask for some free minutes for our open-source-software project. 

If you make a new commit to this PR, I think that it should trigger the travis build.

MikhailRyazanov commented:
Thanks, @DanHickstein, for reviewing and solving the unexpected Travis CI problem!
I don't think that testing this PR further is really needed, since it didn't change any code... And I hope to try something about the default paths soon, so then we'll have a chance to check all the builds. By the way, can we get any automatic macOS testing?

MikhailRyazanov commented:
I've added some technical corrections to the documentation:
1. I can't check the old readthedocs logs, but my local documentation building now started to complain “`WARNING: Duplicate explicit target name: ...`” in several places. So I've switched all (?) inline references to anonymous (with double trailing underscores). Hopefully, this doesn't break anything — I did check some random pages and links, but please also take a look.
2. The Anaconda link https://www.continuum.io/downloads in README apparently is permanently dead (no idea when they've stopped maintaining it, and https://www.continuum.io now redirects to https://www.anaconda.com), so I've replaced it with https://www.anaconda.com/products/individual. Is this the most relevant one?
3. The axes in one of rBasex documentation figures at some point (new Matplotlib version?) became clipped, so I've adjusted the cropping rectangle slightly. For now, seems fine...

DanHickstein commented:
> By the way, can we get any automatic macOS testing

Yes, it looks like travis-ci now supports Windows and macOS testing! So, we can certainly enable Mac testing and maybe also switch to travis for our Windows testing. I have nothing against Appveyor, but I think that we were only using them because Travis didn't support windows testing. One-stop-shopping seems nice. I opened #323 to discuss this. 

> So I've switched all (?) inline references to anonymous (with double trailing underscores). Hopefully, this doesn't break anything

Huh, that's weird. Links in the docs seem to be working based on a few random clicks.

> so I've replaced it with https://www.anaconda.com/products/individual. Is this the most relevant one?

Yes, seems like the best option.

> so I've adjusted the cropping rectangle slightly

Great!

> P.S. Travis now works again. Thanks, Dan! :–)

Awesome!

I realized that the Travis builds actually "cost" at least 10 credits per build minute: https://docs.travis-ci.com/user/billing-overview/

So, I asked for more credits and they gave us 25,000 more for our open source software project. I think that we'll just need to ask for more every few years when they run out.

MikhailRyazanov commented:
>> So I've switched all (?) inline references to anonymous (with double trailing underscores). Hopefully, this doesn't break anything
>
> Huh, that's weird. Links in the docs seem to be working based on a few random clicks.

I in fact can check the old logs (found the link that you gave me) and don't see such warnings in the [previous build](https://readthedocs.org/api/v2/build/14089478.txt). However, it reports `Running Sphinx v1.8.5` ([>2 years old](https://pypi.org/project/Sphinx/1.8.5/), by the way), whereas my local is `Running Sphinx v3.4.3`. I guess, something has changes in the meantime. And surprisingly, other people [complained](https://github.com/sphinx-doc/sphinx/issues/3921) about this already in 2017!
Anyway, if my changes did not break our readthedocs build, then let's keep them just in case.

Should I then merge this PR now to let you work on #323?

stggh commented on README.rst:
```diff
@@ -191,12 +191,12 @@ Citation
 First and foremost, please cite the paper(s) corresponding to the implementation of the Abel transform that you use in your work. The references can be found at the links above.
 
-If you find PyAbel useful in you work, it would bring us great joy if you would cite the project. You can find the DOI for the lastest verison `here <https://dx.doi.org/10.5281/zenodo.594858>`_
+If you find PyAbel useful in you work, it would bring us great joy if you would cite the project. You can find the DOI for the lastest verison `here <https://dx.doi.org/10.5281/zenodo.594858>`__
```
I think the article citation should come first, otherwise authors may cite the zenodo instead of the publication?

DanHickstein replied:
Agreed.

stggh reviewed (commented):
You are a perfectionist @MikhailRyazanov. Good work.
